scripts/pages/pgPeopleLinkDetail.ts

```diff
@@ -0,0 +1,66 @@
+import PgPeopleLinkDetailDesign from 'generated/pages/pgPeopleLinkDetail';
+import LviProperty from 'components/LviProperty';
+import { execute } from 'services/people'
+
+export default class PgPeopleLinkDetail extends PgPeopleLinkDetailDesign {
+    key: string;
+    value: string;
+    data = [];
+    constructor() {
+        super();
+        // Overrides super.onShow method
+        this.onShow = onShow.bind(this, this.onShow.bind(this));
+        // Overrides super.onLoad method
+        this.onLoad = onLoad.bind(this, this.onLoad.bind(this));
+    }
+    initHeader() {
+        this.headerBar.title = this.key;
+    }
+    async fetchData() {
+        try {
+            const res = await execute(this.value);
+            this.data = Object.entries(res).filter((value) => ['string', 'number', 'boolean'].includes(typeof value[1]))
+        } catch (error) {
+            console.log(error)
```

Comment:
You might want to throw your error instead of logging it, so the `Application.onUnhandledError` method we defined in app.ts would trigger for better error handling.